src/platform/sound/manager.js

```diff
@@ -46,52 +43,18 @@ class SoundManager extends EventHandler {
          */
         this._context = null;
 
-        /**
-         * @type {boolean}
-         * @private
-         */
-        this._forceWebAudioApi = options.forceWebAudioApi;
-
-        /**
-         * The function callback attached to the Window events USER_INPUT_EVENTS
-         *
-         * @type {EventListenerOrEventListenerObject}
-         * @private
-         */
-        this._resumeContextCallback = null;
+        // @ts-ignore
```

Comment:
I'd rather we disallowed `@ts-ignore` in the codebase. It appears nowhere else. There must be an alternative...

Reply:
Oh right, I'm so used to working in ts these days... removed.